akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -303,7 +406,7 @@ abstract class UntypedPersistentActor extends UntypedActor with Eventsourced wit
 /**
  * Java API: an persistent actor - can be used to implement command or event sourcing.
  */
-abstract class AbstractPersistentActor extends AbstractActor with PersistentActor with Eventsourced {
+abstract class AbstractPersistentActor extends AbstractActor with Eventsourced {
```

Comment:
Which should indicate that there is no test coverage for the Java API, I'll look into that as well.